index.html

```diff
@@ -377,7 +377,7 @@
                         </dd>
                     <dt><dfn>getCoalescedEvents</dfn></dt>
                         <dd>
-                            <p>Returns a sequence of all <code>PointerEvents</code> that were coalesced into the event. The coalesced events will have the same value as the event in attributes like <code>pointerId</code>, <code>pointerType</code>, <code>isPrimary</code>, and particularly <code>target</code>. But other attributes might be different. The event itself will have the greatest <a href="https://dom.spec.whatwg.org/#dom-event-timestamp">timeStamp</a> ([[!WHATWG-DOM]]) among the events in the sequence and the first event in the sequence will have the smallest timeStamp. None of the events in the returned sequence will have coalesced events. When an event has no coalesed event this function returns an empty sequence.</p>
+                            <p>Returns a sequence of all <code>PointerEvents</code> that were coalesced into the (original) event. The following attributes of the coalesced events will always have the same value as the original event: <code>pointerId</code>, <code>pointerType</code>, <code>isPrimary</code>, and particularly <code>target</code>. But other attributes might be different. The events in the sequence will have increasing <a href="https://dom.spec.whatwg.org/#dom-event-timestamp"><code>timeStamps</code></a> ([[!WHATWG-DOM]]). So the first event will have the smallest <code>timeStamp</code>. Also the original event's <code>timeStamp</code> will be greater than or equal to the last event's <code>timeStamp</code> in the sequence. None of the events in the sequence will have coalesced events. When an event has no coalesed event this function returns an empty sequence.</p>
```

Comment:
I think we should clarify the "global" event ordering too. Without any coalescing, it is assumed that event listeners are fired in the order input events occur. Coalescing adds buffering to it, breaking the assumption. The spec should clarify that despite the buffering, the ordering the "pointermove" handlers are triggered after the corresponding "pointerup" and before the corresponding "pointerdown"/"pointercancel".

I believe all we need is adding a normative test somewhere saying that received events' timestamps are non-decreasing across all pointers. Please correct me if I missed the related spec---I couldn't find any.